config/initializers/omniauth.rb

```diff
@@ -12,7 +12,7 @@
            },
            name: 'CCID'
 
-  provider :developer unless Rails.env.production?
+  provider :developer unless (Rails.env.production? && !ENV['ENABLE_TMP_LOGINS'])
```

Comment:
I think it would be a little more straightforward to test for the positive instead of the negation. 

`provider :developer unless (Rails.env.production? || ENV['ENABLE_TMP_LOGINS'].present?)`